docs/source/tutorials/object_detection.ipynb

```diff
@@ -0,0 +1,357 @@
+   "execution_count": null,
+   "id": "3d70bec6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "example_idx = 120\n",
+    "predictions[example_idx]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d0607aee",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "labels[example_idx]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "06db60bb",
+   "metadata": {},
+   "source": [
+    "We can visualize the bounding boxes and labels side by side using `cleanlabs.object_detection.rank.visualize` function. Here the given labels are in red and the predicted in blue."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "56705562",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "image_path = 'example_images/' + labels[example_idx]['seg_map']\n",
+    "\n",
+    "visualize(image_path, labels[example_idx], predictions[example_idx])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3daff923",
+   "metadata": {},
+   "source": [
+    "# 3. Use cleanlab to find label issues\n",
+    "Based on the given `labels` and model `predictions` from our trained model, cleanlab can quickly help us find examples that contain label errors in the dataset. In object detection, label errors are annotations that are more likely imperfect.\n",
+    "\n",
+    "Examples are imperfect when annotators:\n",
+    "- overlooked an object (missing annotated bounding box),\n",
+    "- chose the wrong class label for an annotated box in the correct location,\n",
+    "- imperfectly annotated the location/edges of a bounding box.\n",
+    "\n",
+    "\n",
+    "For cleanlab, any of these annotation errors should lead to an image with a lower label quality score."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3cceeefb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "num_examples_to_show = 20"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4caa635d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "label_issue_idx = find_label_issues(labels, predictions, return_indices_ranked_by_score=True)\n",
+    "label_issue_idx[:num_examples_to_show]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "638a9dc4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "is_label_issue = find_label_issues(labels, predictions)\n",
+    "is_label_issue[:num_examples_to_show]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5b501dc9",
+   "metadata": {},
+   "source": [
+    "## Label quality scores\n",
+    "The above example shows how to identify which examples have label quality issues. We can also compute scores for each example in the dataset which estimate our confidence that this example has been correctly labeled. These scores range between 0 and 1 with smaller values indicating examples whose annotation seems more likely to be imprefect."
```

Comment:
clarify LQS is useful for prioritizing which images to review, if you have too little time to review all of the label issues.